horovod/spark/torch/remote.py

```diff
@@ -548,3 +550,15 @@ def calculate_loss(outputs, labels, loss_weights, loss_fns, sample_weights=None)
         return loss
 
     return calculate_loss
+
+
+def make_list_if_not_list_fn():
```

Comment:
I would move this to `horovod/spark/common/util.py`.

Reply:
moved.